src/NosCore.Packets/ClientPackets/FDepositPacket.cs

```diff
@@ -0,0 +1,43 @@
+﻿//  __  _  __    __   ___ __  ___ ___  
+// |  \| |/__\ /' _/ / _//__\| _ \ __| 
+// | | ' | \/ |`._`.| \_| \/ | v / _|  
+// |_|\__|\__/ |___/ \__/\__/|_|_\___| 
+// 
+// Copyright (C) 2018 - NosCore
+// 
+// NosCore is a free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using NosCore.Core.Serializing;
+using NosCore.Shared.Enumerations.Items;
+
+namespace NosCore.Packets.ClientPackets
+{
+    [PacketHeader("f_deposit")]
+    public class FDepositPacket : PacketDefinition
+    {
+        [PacketIndex(0)]
+        public PocketType Inventory { get; set; }
+
+        [PacketIndex(1)]
+        public byte Slot { get; set; }
+
+        [PacketIndex(2)]
+        public short Amount { get; set; }
+
+        [PacketIndex(3)]
+        public byte NewSlot { get; set; }
+
+        [PacketIndex(4)]
+        public byte? Unknown { get; set; }
```

Comment:
is it -1 or just not displayed ? in that case it would be optional

Reply:
is it always -1